circle.yml

```diff
@@ -20,3 +20,4 @@ deployment:
     branch: master
     commands:
       - NODE_ENV=prod npm run build
+      - npm publish
```

Comment:
@stulees The `npm publish` call implicitly calls `npm prepublish` before publishing. I'd mistakenly published once without the prepublish script in place, and it didn't deploy `dist` because I hadn't built it. So prepublish prevents that situation from occurring.